### ISSUE TITLE
ZFnet  inferences: weird results

### ISSUE THREAD

**Issue by 3outeille** [osmr/imgclsmob#51]
Hey,

I tried to use the ZFNet pretrained model to see how this one perform. I add softmax function and here is the weird result I got: 

![result](https://user-images.githubusercontent.com/47445085/73963992-a0d7f280-4911-11ea-9ddf-e6ad01f50389.png)

I really wonder where does the problem come from. My code doesn't seem to have any error. Link to the repo : https://github.com/3outeille/Research-Paper-Summary/blob/master/src/architecture/zfnet/pytorch/zfnet_pytorch.ipynb

```python
imagenet_mean = np.array([104., 117., 124.], dtype=np.float32)
fig2 = plt.figure(figsize=(30,10))

for i, image in enumerate(imgs):
    img = cv2.resize(image.astype(np.float32), (224, 224))
    img -= imagenet_mean
    img = img.reshape((1, 3, 224, 224))
    img = torch.from_numpy(img)
    
    probs = torch.nn.Softmax(dim=-1)(net(img))
    probs = probs.detach().numpy()
```
Does it mean that there was a bug during ZFNet training ?

**Comment by osmr** [comment]
No, it means that there is a bug in the inference script. You need to use the ImageNet-1K standard preprocessing. I will clarify the answer later. See https://github.com/pytorch/examples/blob/master/imagenet/main.py.

**Comment by 3outeille** [comment]
Okay, I'll wait for clarification.

**Comment by 3outeille** [comment]
I follow pytorch standard preprocessing steps. It worked. Thanks a lot.

**Comment by osmr** [comment]
Also look at this sample code: https://github.com/osmr/imgclsmob/blob/master/examples/demo_pt.py.
